Specs/Scene/ModelExperimental/InstancingPipelineStageSpec.js

```diff
@@ -519,7 +613,7 @@ describe("Scene/ModelExperimental/InstancingPipelineStage", function () {
         CesiumMath.EPSILON8
       );
 
-      // matrices are stored as 3 vec4s, so this is
+      // Matrices are stored as 3 vec4s, so this is
       // 25 matrices * 12 floats/matrix * 4 bytes/float = 1200
       const matrixSize = 1200;
       // 25 floats
```

Comment:
(bottom of the file) -- this test suite should be marked as `WebGL`, looks like we forgot that when we first wrote these tests.